beast/physicsmodel/tests/test_dust_prior_weights.py

```diff
@@ -19,28 +18,28 @@ def test_dust_prior_weights():
     prior_weights = PriorWeightsDust(av_vals, av_model, rv_vals, rv_model, fA_vals, fA_model)
 
     # test if priors are set correctly when creating an object of the class PriorWeightsDust
-    np.testing.assert_allclose(prior_weights.av_priors,np.full((len(av_vals)), 1.0))
-    np.testing.assert_allclose(prior_weights.rv_priors,np.full((len(rv_vals)), 1.0))
-    np.testing.assert_allclose(prior_weights.fA_priors,np.full((len(fA_vals)), 1.0))
+    np.testing.assert_allclose(prior_weights.av_priors,np.full((len(av_vals)), 1.0), err_msg="A problem occurred with setting the A(V) priors when creating a PriorWeightsDust object. They should be set to flat by default.")
+    np.testing.assert_allclose(prior_weights.rv_priors,np.full((len(rv_vals)), 1.0), err_msg="A problem occurred with setting the R(V) priors when creating a PriorWeightsDust object. They should be set to flat by default.")
+    np.testing.assert_allclose(prior_weights.fA_priors,np.full((len(fA_vals)), 1.0), err_msg="A problem occurred with setting the f_A priors when creating a PriorWeightsDust object. They should be set to flat by default.")
 
 
     # Test A(V) priors
     # test default: model=flat
     prior_weights.set_av_weights()
-    np.testing.assert_allclose(prior_weights.av_priors,np.full((len(av_vals)), 1.0))
+    np.testing.assert_allclose(prior_weights.av_priors,np.full((len(av_vals)), 1.0), err_msg="A problem occurred while setting the A(V) priors. They should be flat by default.")
 
     # test other models (with random input values)
     expected_lognormal_av = [0., 2.00684868, 1.57828555, 1.09756182, 0.7677122, 0.54959176]
     prior_weights.set_av_weights(model={"name" : "lognormal", "max_pos" : 2.0, "sigma" : 1.0})
-    np.testing.assert_allclose(prior_weights.av_priors, expected_lognormal_av)
+    np.testing.assert_allclose(prior_weights.av_priors, expected_lognormal_av, err_msg="A problem occurred while setting the A(V) priors to lognormal.")
 
     expected_two_lognormal_av = [0.0, 5.962474, 3.07642414e-02, 4.46347150e-03, 1.60916384e-03, 6.89125670e-04]
     prior_weights.set_av_weights(model={"name" : "two_lognormal", "max_pos1" : 0.2, "max_pos2": 2.0, "sigma1" : 1.0, "sigma2" : 0.2, "N1_to_N2" : 1.0 / 5.0})
-    np.testing.assert_allclose(prior_weights.av_priors, expected_two_lognormal_av)
+    np.testing.assert_allclose(prior_weights.av_priors, expected_two_lognormal_av, err_msg="A problem occurred while setting the A(V) priors to two_lognormal."))
```

Comment:
There is an extra `)` at the end of this line.  That is causing the tests to fail.  Same issue for at least a couple of the other similar lines.

Reply:
Argh, stupid mistake! Sorry about that :)